packages/node_modules/@node-red/editor-client/src/js/ui/workspaces.js

```diff
@@ -904,6 +904,13 @@ RED.workspaces = (function() {
             }
         },
         refresh: function() {
+            var workspace = RED.nodes.workspace(RED.workspaces.active());
+            if (workspace) {
+                document.title = `${documentTitle} : ${workspace.label}`;
+            } else {
+                var subflow = RED.nodes.subflow(RED.workspaces.active());
+                document.title = `${documentTitle} : ${subflow.name}`;
```

Comment:
```suggestion
                if (subflow) {
                    document.title = `${documentTitle} : ${subflow.name}`;
                } else {
                    document.title = documentTitle
                }
```

Adding an extra guard just to be safe